README.md

```diff
@@ -262,15 +266,35 @@ Print out the Nile version
 nile version
 ```
 
-### `debug`
+### `status`
 
-Use locally available contracts to make error messages from rejected transactions more explicit.
+Prints the current status of a transaction.
 
 ```sh
-nile debug <transaction_hash> [CONTRACTS_FILE, NETWORK]
+nile status <transaction_hash> [CONTRACTS_FILE, NETWORK, TRACK/DEBUG]
```

Comment:
```suggestion
nile status <transaction_hash> [CONTRACTS_FILE, NETWORK, TRACK/DEBUG]
```
these examples are inconsistent with the recent changes (`--track/--debug`). i'm still a bit confused by this syntax. is it common to refer to flags like this?

Reply:
because i see it used for other flags/params as well

Reply:
I'm honestly not sure if it's common. I'll update the examples to something less confusing